paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormViewModel.kt

```diff
@@ -251,4 +193,26 @@ internal class FormViewModel @Inject internal constructor(
             !hiddenIds.contains(it)
         }
     }
+
+    companion object {
+        fun getElements(
+            context: Context,
+            formFragmentArguments: FormFragmentArguments,
+            lpmResourceRepository: ResourceRepository<LpmRepository>,
+            addressResourceRepository: ResourceRepository<AddressRepository>
+        ) = TransformSpecToElements(
+            addressResourceRepository = addressResourceRepository,
+            initialValues = formFragmentArguments.getInitialValuesMap(),
+            amount = formFragmentArguments.amount,
+            saveForFutureUseInitialValue = formFragmentArguments.showCheckboxControlledFields,
+            merchantName = formFragmentArguments.merchantName,
+            context = context,
+            shippingValues = formFragmentArguments.shippingDetails
+                ?.toIdentifierMap(formFragmentArguments.billingDetails)
+        ).transform(
+            requireNotNull(
+                lpmResourceRepository.getRepository().fromCode(formFragmentArguments.paymentMethodCode)
+            ).formSpec.items
+        )
+    }
```

Comment:
Should this transformer be injected in `FormViewModel` instead? Then, we could directly have the view model emit the list of elements, instead of having to go via `LaunchedEffect` in the Composable.